### ISSUE TITLE
How to fetch destinations from a specific BTP subaccount different from the current one?

### ISSUE THREAD

**Issue by ptesny** [SAP/cloud-sdk-js#3626]
How do I tell cloud-sdk to go and fetch destinations from a specific BTP subaccount and different from the one used for users authentication ?

**Comment by marikaner** [comment]
I don't think this is currently intended in the SAP Cloud SDK. You can [change the selection strategy](https://sap.github.io/cloud-sdk/api/v3/variables/sap_cloud_sdk_connectivity.DestinationSelectionStrategies.html) and adjust whether to use the subscriber or provider. I assume this is not enough for your case?
Could you please describe your use case?

**Comment by ptesny** [comment]
@marikaner, indeed, as you have guessed, my use case is not a classic [multi-tenancy](https://sap.github.io/cloud-sdk/docs/js/features/connectivity/destinations#multi-tenancy) one-to-many relationship, with a single instance of a provider and multiple subscribers (consumers, data sources, etc);

What I am aiming at is to be able to **share** a number static resources (destinations, html5 repo, cloud connectors) that reside or are hooked into a single **master** BTP subaccount among a number of providers for enhanced resilience. 
There can of course be mutlitenancy for each of the providers....and consumers can subscribe to multiple providers as well;

What I would like to have is a shared and static pool of destinations....and if dynamic changes were required then the off-spring destinations would be set up on the fly as instance level or inline destinations....

**Comment by jjtang1985** [comment]
This issue has been automatically closed due to inactivity.
Please reopen a new one, if needed.
Thank you!
